### PR TITLE
docs(readme): add a TL;DR

### PR DESCRIPTION
**Pull request by EisenbergEffect**
# Description

I saw some community feedback that a tl;dr; would be appropriate in the readme, so I put this together.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,14 @@
 
 This is the FAST monorepo, containing web component packages, tools, examples, and documentation. FAST tech can be used à la carte or as a suite to build enterprise-grade websites, applications, components, design systems, and more.
 
+**tl;dr;**
```
Comment (janechu) [inline README.md:20]: nit: should be TL;DR

File: README.md [hunk 2]
```diff
@@ -17,6 +17,14 @@
...
+
+* Create reusable UI components based on web component standards.
+* Use our standard library of pre-built web components in your apps.
+* Choose between two *adaptive* design systems: Fluent Design and FAST Frame.
```
Comment (janechu) [inline README.md:24]: There may soon only be FAST design system as msft may move to Fluent UI

**Comment by janechu** [comment]
Just a note, changed your PR title from "doc" to "docs" to pass title linting

File: README.md (hunk 2 above)
Comment (EisenbergEffect) [reply README.md:24]: I'm thinking that I could turn these into links and link out to the fluent repo for that since it's still a design system built by our team.

File: README.md (hunk 1 above)
Comment (EisenbergEffect) [reply README.md:20]:
```suggestion
**TL;DR**
```
